src/ros_license_toolkit/package.py

```
@@ -263,11 +263,12 @@ def get_copyright_file_contents(self) -> str:
             cpr_str += f"\nLicense: {pkg_license.id}\n"
             assert pkg_license.license_text_file, \
                 "License text file must be defined."
-            with open(os.path.join(
-                self.abspath,
-                pkg_license.license_text_file),
-                encoding="utf-8"
-            ) as f:
+            license_path = os.path.join(self.abspath,
+                                        pkg_license.license_text_file)
+            assert os.path.exists(license_path), \
+                ('Cannot create copyright file. '
+                 f'File {license_path} does not exist.')
```

Please raise an exception here. Assertions are not for production